src/components/organisms/AdminVenueView/AdminVenueView.tsx

```diff
@@ -72,7 +78,12 @@ export const AdminVenueView: React.FC = () => {
           {renderAdminVenueTabs}
         </Nav>
       </div>
-      {selectedTab === AdminVenueTab.spaces && <div>Spaces</div>}
+      {selectedTab === AdminVenueTab.spaces && (
+        <Spaces
+          venue={venue as Venue_v2}
+          onClickNext={() => setSelectedTab(AdminVenueTab.timing)}
```

Comment:
useCallback

Reply:
Fixed in: https://github.com/sparkletown/sparkle/pull/1583/commits/f8fa0c67c34ac626919737b4bbd112b4bd715553